types/fundamental-react/lib/Tile/Tile.d.ts

```diff
@@ -1,36 +1,39 @@
+import * as React from "react";
+
 export type TileProps = {
-    active?: boolean,
-    backgroundImage?: string,
-    className?: string,
-    disabled?: boolean,
-    disableStyles?: boolean,
-    productTile?: boolean,
-    tabIndex?: number,
-    onClick?: (...args: any[]) => any
+    active?: boolean;
+    backgroundImage?: string; //TODO does not work yet
+    className?: string;
+    disabled?: boolean;  //TODO does not work yet
+    isDouble?: boolean;
+    size?: "s";
+    onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
 } & { [x: string]: any };
 
-export type TileActionsProps = {
+export type TileContentProps = {
     className?: string;
+    twoColumns?: boolean,
 } & { [x: string]: any };
 
-export type TileContentProps = {
-    title: string,
-    className?: string,
-    headingLevel?: 2 | 3 | 4 | 5 | 6;
-    productTile?: boolean,
-    titleProps?: any
+export type TileFooterProps = {
+    className?: string;
 } & { [x: string]: any };
 
-export type TileMediaProps = {
+export type TileHeaderProps = {
     className?: string;
-    productTile?: boolean;
+    subtitle?: string;
+    //TODO subTitleProps?: https://sap.github.io/fundamental-react/?path=/docs/component-api-tile--primary#properties
```

Comment:
https://github.com/SAP/fundamental-react/blob/v0.13.1/src/Tile/_TileHeader.js#L30

Here you see the usage
these are props for the react div (`React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>`)
https://stackoverflow.com/questions/51835611/specify-specific-props-and-accept-general-html-props-in-typescript-react-app